SB-1.md

```diff
@@ -0,0 +1,76 @@
+# SB-1 (AMQP Backed Implementation)
+
+**Status**: Under Review
+
+This proposal introduces a method of using a message queue (Anything implementing
+AMQP 0.9.1, focusing on RabbitMQ) rather than the internal, custom queue.
```

Comment:
`AMQP 0.9.1` Why the specific version?

Reply:
RabbitMQ implements 0.9.1. AMQP 1.0 completely changed the protocol, and the model of many message brokers didn't fit super well with it, so they never switched over. AMQP 0.9.1 is the most widely supported and does what we need.

I should add something like that to the proposal...